### DISCUSSION TITLE
rex board isometric style example

### DISCUSSION THREAD

**Discussion by crapthings** [rexrainbow/phaser3-rex-notes#245] [Q&A]
is there an example for isometric?

**Comment by rexrainbow** [comment]
Like [this one](https://codepen.io/rexrainbow/pen/bGgRVeM)?

**Reply by crapthings** [reply]
yeah~~~~

so how to put texture on it

I want to make kinda monopoly/大富翁, should i make tilemap with tilemap editor or just put background on board?

**Reply by crapthings** [reply]
the board is like a square

four road on side, and the outside are buildings

the center around the road will be another game object

something like this

![image](https://user-images.githubusercontent.com/1147704/153988757-4a4d9e24-c53a-4c0f-9652-890e710e5abb.png)

I think I can start with this one, and change type to isometric

https://codepen.io/rexrainbow/pen/WWJxZL

**Reply by rexrainbow** [reply]
> yeah~~~~
> 
> so how to put texture on it
> 
> I want to make kinda monopoly/大富翁, should i make tilemap with tilemap editor or just put background on board?

You can create image/sprite or any kind of game object, then [add](https://rexrainbow.github.io/phaser3-rex-notes/docs/site/board/#add-chess) it to board at (tileX, tileY, tileZ) logical position.
